requirements/default.txt

```diff
@@ -139,9 +139,6 @@ urlwait==0.4 \
 pytest==5.2.2 \
     --hash=sha256:27abc3fef618a01bebb1f0d6d303d2816a99aa87a5968ebc32fe971be91eb1e6 \
     --hash=sha256:58cee9e09242937e136dbb3dab466116ba20d6b7828c7620f23947f37eb4dae4
-pytest-wholenodeid==0.2 \
-    --hash=sha256:6bc4edaf560c8b45ac4669bb8b99d8fbbb7f6e5af8dc7727200cd4d3ef4668c2 \
-    --hash=sha256:bdcb4df88ca182a19752ef95150c0881deaafc2c593ef67c61515894e5309206
```

Comment:
I'm no longer maintaining this library. It also has some minor issues with it for certain kinds of test definitions--the details escape me. Anyhow, time to stop using it.